PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/README.md

```diff
@@ -13,3 +13,11 @@ The Java Code Quality Analyzer, is a plugin designed to improve the quality of J
 
 ## User Interface
 - **Telemetry Configuration Panel** : This space allows users to enable or disable telemetry if desired.
+
+
+## Rules
+1. #### Using Connection Strings to build Azure Service Clients
```

Comment:
Need to rebase and adjust this section accordingly.

Reply:
done:)